packages/image_picker/image_picker/example/android/app/build.gradle

```diff
@@ -36,6 +36,7 @@ android {
         applicationId "io.flutter.plugins.imagepicker.example"
         minSdkVersion 16
         targetSdkVersion 28
+        multiDexEnabled true
```

Comment:
did it hit the 64K limit?

Reply:
Yea it did. And same for the other plugins that I also added the property to in this PR